Papr/Commons/Constants.swift

```diff
@@ -117,16 +117,30 @@ enum Constants {
 
             static var squareAndArrowDownMedium: UIImage {
                 if #available(iOS 13.0, *) {
-                    return UIImage(systemName: "square.and.arrow.down", withConfiguration: symbolConfigurationMedium)!
+                    return UIImage(systemName: "arrow.down", withConfiguration: symbolConfigurationMedium)!
                 }
-                return UIImage(imageLiteralResourceName: "down-black")
+                return UIImage(imageLiteralResourceName: "arrow.down")
             }
 
             static var squareAndArrowDownSmall: UIImage {
                 if #available(iOS 13.0, *) {
-                    return UIImage(systemName: "square.and.arrow.down", withConfiguration: symbolConfigurationSmall)!
+                    return UIImage(systemName: "arrow.down", withConfiguration: symbolConfigurationSmall)!
                 }
-                return UIImage(imageLiteralResourceName: "down-black")
+                return UIImage(imageLiteralResourceName: "arrow.down")
+            }
+
+            static var doneWhiteSmall: UIImage {
+                if #available(iOS 13.0, *) {
+                    return UIImage(systemName: "checkmark", withConfiguration: symbolConfigurationSmall)!.withTintColor(.white)
+                }
+                return UIImage(imageLiteralResourceName: "done")
+            }
+
+            static var addSmall: UIImage {
```

Comment:
nit: Can we rename this to `addMedium`?  (b/c it is using a medium configuration)

Reply:
Completely my bad! Will do this tonight!